frontend/src/scenes/persons/personsLogic.test.ts

```diff
@@ -108,6 +108,16 @@ describe('personsLogic', () => {
                 })
         })
 
+        it('loads a person where id includes +', async () => {
+            await expectLogic(logic, () => {
+                logic.actions.loadPerson('+')
```

Comment:
Frustrating that if someone breaks the code you get a very opaque error message here.

The mock receives a single space when the test calls it with the code re-broken. Would be better to grab the call to `api.get` and assert on it including %2B and not + for the future traveller...

But this is better than before :)

Reply:
I'm going to get this in as is as I'm a little stretched atm, sorry 💣